src/pages/executeFlow/Execute.tsx

```diff
@@ -130,273 +156,306 @@ function ExecuteFlow({
-            <h2 className="tds-headline-02">{iri}</h2>
-          </div>
-          {mode === "initial" && (
-            <div className={styles.contentContainer__parameterContainer}>
-              <tds-button
-                class={styles.contentContainer__parameterContainer__btn}
-                onClick={() => setMode("newParameter")}
-                type="button"
-                size="sm"
-                variant="primary"
-                fullbleed
-                text="Create new parameter"
-              ></tds-button>
-              <span>or</span>
-              <tds-dropdown
-                class={styles.contentContainer__parameterContainer__dropdown}
-                name="dropdown"
-                placeholder="Choose parameter"
-                size="md"
-                open-direction="auto"
-              >
-                {parameters.map((param, index) => {
-                  return (
-                    <tds-dropdown-option
-                      value={param.name}
-                      key={index}
-                      onClick={() => selectParameter(param)}
-                    >
-                      {param.name}
-                    </tds-dropdown-option>
-                  );
-                })}
-              </tds-dropdown>
+        <hr className="divider" />
+        <div className={styles.headerContainer}>
+          <h3 className="tds-headline-03" style={{ marginBottom: "16px" }}>
+            Execution flow
+          </h3>
+          <div className={styles.headerContainer__detailsContainer}>
+            <div className="tds-detail-02">
+              Graph name:{" "}
+              <span className="bold"> Placeholder for graph name</span>
             </div>
-          )}
-          {mode === "newParameter" && (
-            <div className={styles.contentContainer__newParameterContainer}>
-              <tds-text-field
-                placeholder="Placeholder"
-                label="Parameter name"
-                label-position="outside"
-                value={creatingNewParameter.name}
-                onInput={(e: { currentTarget: { value: string } }) =>
-                  setCreatingNewParameter({
-                    ...creatingNewParameter,
-                    name: e.currentTarget.value,
-                  })
-                }
-              />
-              <tds-textarea
-                rows="20"
-                label="JSON"
-                value={creatingNewParameter.value}
-                label-position="outside"
-                placeholder="Placeholder"
-                onInput={(e: { currentTarget: { value: string } }) =>
-                  setCreatingNewParameter({
-                    ...creatingNewParameter,
-                    value: e.currentTarget.value,
-                  })
-                }
-              ></tds-textarea>
+            <div className="tds-detail-02">
+              User: <span className="bold">Placeholder for user </span>
```

Comment:
What do we do with this? Can we get username/name from session?

Reply:
I have removed it for now, for both the graph name we need an api, let's add it back later